src/server/MCFadapter.js

```diff
@@ -5,41 +5,67 @@ export class MCFAdapter {
   pool = null
   binsHash = {}
 
-  setPool = dbObj => {
+  setPool = (dbObj, mongodbObj) => {
```

Comment:
Technically once you remove the postgres code, it's incorrect to call this function `setPool`. 
It's better to change it to `setDB` or something.